src/index.css

```diff
@@ -40,8 +40,11 @@ body::-webkit-scrollbar-thumb {
   ); /* Chrome 10-25, Safari 5.1-6 */
   background: linear-gradient(
     to right bottom,
-    #7f7fd5,
+    /* #7f7fd5,
```

Comment:
what is the need of these

Reply:
I have commented that out so that previous color do not get lost.